pages/common/quarto.md

```diff
@@ -0,0 +1,36 @@
+# quarto
+
+> An open-source scientific and technical publishing system built on Pandoc.
+> More information: <https://quarto.org/>.
+
+- Create a Quarto project:
+
+`quarto create-project {{destination_directory}}`
+
+- Create a new website project:
+
+`quarto create-project {{destination_directory}} --type website`
+
+- Create a new book project:
+
+`quarto create-project {{destination_directory}} --type book`
+
+- Render R Markdown file to HTML:
+
+`quarto render {{input.rmd}} --to html`
+
+- Render Quarto file to HTML:
+
+`quarto render {{input.qmd}} --to html`
+
+- Render Jupyter notebooks to HTML:
+
+`quarto render {{input.ipynb}} --to html`
+
+- Render document to PDF:
```

Comment:
Which kind of document is this? The command itself is exactly the same as rendering a Quarto file.

Reply:
It is a Jupyter Notebook.

Reply:
Sorry, wrong line. I mean the `input.qmd` file.

Reply:
It is a Quarto document. Very similar to an R Markdown document: https://quarto.org/docs/get-started/hello/rstudio.html

Reply:
Does converting to `docx` and `pdf` only work for Quarto documents?